var/spack/repos/builtin/packages/netcdf-c/netcdfc-hdf5-link-mpi.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/liblib/CMakeLists.txt b/liblib/CMakeLists.txt
+index aa3a842d..691902c2 100644
+--- a/liblib/CMakeLists.txt
++++ b/liblib/CMakeLists.txt
+@@ -50,6 +50,7 @@ ADD_LIBRARY(netcdf nc_initialize.c ${LARGS} )
+ 
+ IF(MPI_C_INCLUDE_PATH)
+     target_include_directories(netcdf PUBLIC ${MPI_C_INCLUDE_PATH})
++    target_link_libraries(netcdf MPI::MPI_C)
```

Comment:
These patches appear to be fundamental - do you know if there are any PRs for `netcdf-c` adding this functionality?

Reply:
Sorry, I must have missed this comment last week. Yes, there are, I made a PR implementing the relevant changes in netcdf around when I made this PR. FWIW the include change was not included in my changes as it seems as if there is some debate as to that point between netcdfc and hdf5.